velox/expression/tests/CustomTypeTest.cpp

```diff
@@ -109,13 +125,36 @@ class FromFancyIntFunction : public exec::VectorFunction {
                 .build()};
   }
 };
+
+struct FancyIntT {
+  using type = std::shared_ptr<FancyInt>;
+  static constexpr const char* typeName = "fancy_int";
+};
+using TheFancyInt = CustomType<FancyIntT>;
```

Comment:
```
nit: s/TheFancyInt/FancyIntType/
```

Reply:
That name is taken. FancyIntType is an instance of velox::Type that describes the type. TheFancyInt is C++ type of the values.